src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -237,15 +238,8 @@ export default class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang
   }
 
   get flow(): string {
-    const {
-      flows,
-      resources,
-    } = this as { flows: IFlow[], resources: IResource[] }
     return JSON.stringify(
-      {
-        flows,
-        resources,
-      },
+      omit(this.activeFlowContainer, 'isCreated'),
```

Comment:
The purpose of this change is to make sure you get a valid flow container from the export panel so we can potentially demo an export/import loop